apps/bridge-dapp/src/utils/downloadNotes.ts

```diff
@@ -0,0 +1,24 @@
+import { downloadString } from '@webb-tools/browser-utils';
+import { Note } from '@webb-tools/sdk-core';
+import React from 'react';
+
+/**
+ * Convert notes to strings and download them as json file
+ * @param notes the notes to download
+ * @returns boolean - true if the download was successful
+ */
+export const downloadNotes = (notes: Note[]) => {
```

Comment:
Can we start adding explicit return types to functions too? Especially if we're going to document it.